src/fsharp/vs/service.fs

```diff
@@ -760,6 +775,147 @@ type TypeCheckInfo
 
         let FindRecordFieldsInEnv(plid, residue) = GetClassOrRecordFieldsEnvironmentLookupResolutions(line, loc, plid, residue)
 
+        let getDeclaredItems isInRangeOperator =
```

Comment:
this function is pretty much unchanged code that use to start at the line 787 - this is common logic that looks for declarations. Now it is used in two places to I've moved it to the function